app/export_config/generate_fund_round_config.py

```diff
@@ -172,4 +176,108 @@ def generate_config_for_round(round_id):
+    )
+
+    if Config.GENERATE_LOCAL_CONFIG:
+        generate_default_assessment_mappings(fund_config, round_config)
+
+
+def generate_default_assessment_mappings(fund_config, round_config):
+    # The following config is not tested for production use
+    # It is generated to make local testing easier - you can add an application to fab and export it with a basic
+    # auto-generated assessment config.
+    # Each form is a sub-critiera, each page a theme. Half scored, half unscored.
+    # The output in the assessment_store folder needs to be added to the
+    # assessment_mapping_fund_round file in assessment-store
+    fund_id = fund_config["id"]
+    round_id = round_config["id"]
+    fund_short_name = fund_config["short_name"]
+    round_short_name = round_config["short_name"]
+    fund_round = f"{str.upper(fund_short_name)}{str.upper(round_short_name)}"
+    fund_round_ids = f"{fund_id}:{round_id}"
+
+    scored = []
+    unscored = []
+    i = 0
+    sections = db.session.query(Section).filter(Section.round_id == round_id).order_by(Section.index).all()
+    for section in sections:
+        i += 1
+        type_of_criteria = "scored" if i % 2 == 0 else "unscored"  # do a random half scored and unscored
+        criteria = {
+            "id": human_to_kebab_case(section.name_in_apply_json["en"]),
+            "name": section.name_in_apply_json["en"],
+            "sub_criteria": [],
+        }
+        if type_of_criteria == "scored":
+            # half the sections will be scored, divide the weighting between them
+            criteria["weighting"] = 1 / (len(sections) / 2)
+            scored.append(criteria)
+        else:
+            unscored.append(criteria)
+
+        for form in section.forms:
+            sc = {
+                "id": form.runner_publish_name,
+                "name": form.name_in_apply_json["en"],
+                "themes": [],
+            }
+            for page in form.pages:
+                if page.display_path == "summary":
+                    continue
+                theme = {
+                    "id": human_to_kebab_case(page.name_in_apply_json["en"]),
+                    "name": page.name_in_apply_json["en"],
+                    "answers": [],
+                }
+                for component in page.components:
+                    if component.type in READ_ONLY_COMPONENTS:
+                        continue
+                    answer = {
+                        "field_id": component.runner_component_name,
+                        "form_name": form.runner_publish_name,
+                        "field_type": component.type.name,
+                        "presentation_type": form_json_to_assessment_display_types.get(component.type.name, "text"),
+                        "question": component.title,
+                    }
+                    theme["answers"].append(answer)
+                sc["themes"].append(theme)
+            criteria["sub_criteria"].append(sc)
+
+    temp_assess_output = {
```

Comment:
@srh-sloan cant we have these as JSON templates ?

Reply:
What do you mean?